prime-router/docs/design/design/RS-Extention-Registry.yml

```diff
@@ -0,0 +1,5930 @@
+              valueType: String
+              hl7Use: EI.4
+    HD:
+      Example:
+        "identifier": [
+          {
+            "extension": [
+              {
+                "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
+                "valueString": "HD.1"
+              }
+            ],
+            "value": "MN Public Health Lab"
+          },
+          {
+            "extension": [
+              {
+                "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
+                "valueString": "HD.2,HD.3"
+              }
+            ],
+            "type": {
+              "coding": [
+                {
+                  "extension": [
+                    {
+                      "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/code-index-name",
+                      "valueString": "identifier"
+                    }
+                  ],
+                  "system": "http://terminology.hl7.org/CodeSystem/v2-0301",
+                  "code": "ISO"
+                }
+              ]
+            },
+            "system": "urn:ietf:rfc:3986",
+            "value": "2.16.840.1.114222.4.1.10080"
+          }
+        ]
+      Extension:
+        - id: hl7v2Field
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field"
+          valueType: String
+          hl7Use: NA
+          description: Identifies the HL7 origin/destination of the data.
+- Resource: Any
+  FHIR-Type: CodeableConcept
+  HL7-Type:
+    CWE:
+      Example:
+        "valueCodeableConcept": {
+          "extension": [
+            {
+              "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/coding-system-oid",
+              "valueOid": "urn:oid:1.22.333.4444"
+            },
+            {
+              "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
+              "valueString": "PID.39"
+            }
+          ],
+          "coding": [
+            {
+              "extension": [
+                {
+                  "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding",
+                  "valueString": "coding"
+                },
+                {
+                  "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding-system",
+                  "valueString": "HL70171"
+                }
+              ],
+              "version": "2.5.1",
+              "code": "S",
+              "display": "Seminole"
+            }
+          ],
+          "text": "TEST"
+        }
+      Extension:
+        - id: coding-system-oid
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/coding-system-oid"
+          valueType: OID
+          hl7Use: CWE.14
+        - id: cwe-coding
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/cwe-coding"
+          valueType: String
+          hl7Use: NA
+          constant: "coding"
```

Comment:
Can't this also be `alt-coding` etc.?

Reply:
yep, and I was missing `secondary-alt-coding` as an option. Added both